rhasspywake_porcupine_hermes/__init__.py

```diff
@@ -73,19 +72,27 @@ def __init__(
         self.sample_width = sample_width
         self.channels = channels
 
-        # Queue of WAV audio chunks to process (plus site_id)
-        self.wav_queue: queue.Queue = queue.Queue()
+        self.audio_buffers = dict()
+        self.first = dict()
+        self.chunks = dict()
+        self.unpacked_chunks = dict()
+        self.wav_queues = dict()
+        self.porcupines = dict()
+        self.chunk_sizes = dict()
+        self.chunk_formats = dict()
+        for site_id in self.site_ids:
```

Comment:
I wonder if this includes the site of the instance as well as the additional site ids that are specified in the text input box. No idea, just making sure.

Reply:
Yes, it includes all site_id's.
At least, it did in my setup which is 1 base and 2 esp32 sat's